src/pages/saksbehandling/steg/beregningOgSimulering/simulering/simulering.tsx

```diff
@@ -45,34 +45,47 @@ const Utbetalingssimulering = (props: { simulering: Simulering }) => {
                     )}
                 </span>
             </Element>
-            {gruppertSimuleringsperioder.map((gruppe) => {
-                return pipe(
-                    combineOptions(arr.head(gruppe), arr.last(gruppe)),
-                    Option.fold(
-                        () => ({
-                            tittel: '?',
-                            beløp: 0,
-                            key: '?',
-                        }),
-                        ([head, last]) => ({
-                            tittel: `${formatMonthYear(head.fraOgMed, intl)} - ${formatMonthYear(last.tilOgMed, intl)}`,
-                            beløp: head.bruttoYtelse,
-                            key: head.fraOgMed + head.tilOgMed,
-                        })
-                    ),
-                    ({ tittel, beløp, key }) => (
-                        <Element tag="h3" className={classNames(styles.periodeoverskrift, styles.linje)} key={key}>
-                            <span>{tittel}</span>
-                            <span>
-                                {formatCurrency(intl, beløp, {
-                                    numDecimals: 0,
-                                })}{' '}
-                                i mnd
-                            </span>
-                        </Element>
-                    )
-                );
-            })}
+            {pipe(
+                props.simulering.perioder,
+                groupWhile(
+                    (curr, prev) =>
+                        DateFns.differenceInCalendarMonths(
+                            DateFns.parseISO(curr.fraOgMed),
+                            DateFns.parseISO(prev.tilOgMed)
+                        ) <= 1
+                ),
+                arr.map((gruppe) => {
+                    return pipe(
+                        combineOptions(arr.head(gruppe), arr.last(gruppe)),
+                        Option.fold(
+                            () => ({
+                                tittel: '?',
+                                beløp: 0,
+                                key: '?',
+                            }),
```

Comment:
Dette borde väl inte ske i praxis?

Reply:
Det stemmer. Jeg syns likevel det er bedre å ha en viss form for håndtering av det enn at vi får `undefined`-feil.